recipes/asfernandes-cloop/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import functools
+import os
+
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class CloopConan(ConanFile):
+    name = "asfernandes-cloop"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/asfernandes/cloop"
+    description = "Cross Language Object Oriented Programming"
+    topics = ("parser", "idl")
+    license = "???"
```

Comment:
This project uses the [IDPL](https://firebirdsql.org/en/initial-developer-s-public-license-version-1-0/). This license does not have a SPDX identifier. What should it be named here?

Reply:
Amazing, we are using Unlicensed when there is no license or is using Public Domain. But here is a different case, we just don't have a short name. IDPL is very close to [Interbase-1.0](https://spdx.org/licenses/Interbase-1.0.html), only the last paragraph is different. I would suggest using `Developer's Public License Version 1.0` for now and adding a # TODO for later. I see the developer involved is Brazilian (whata luck), I'll send an email asking him about the license.

Reply:
@uilianries found it: https://github.com/conan-io/conan-center-index/pull/4928/files#r596208293

Reply:
Thank you @SSE4! I just the email now and pointed this thread.

Reply:
Thanks for looking into this. FirebirdSQL is also licensed with IDPL, so it will prove to be useful for that.